configs/lad/lad_r50_paa_r101_fpn_coco_1x.py

```diff
@@ -123,10 +123,8 @@
         nms=dict(type='nms', iou_threshold=0.6),
         max_per_img=100))
 train_dataloader = dict(batch_size=8, num_workers=4)
-optim_wrapper = dict(optimizer=dict(lr=0.01))
+optim_wrapper = dict(type='AmpOptimWrapper', optimizer=dict(lr=0.01))
```

Comment:
Is it verified that using AMP could train the model correctly?

Reply:
Training can be started normally, and I am verifying its accuracy.